### PR TITLE
[keyboard] add jaykeeb jk65

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

add support 65% layout for any keyboard case exiting

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jaykeeb/jk65/info.json [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+        "device_version": "1.0.0",
+        "pid": "0x7765",
+        "vid": "0x414C"
+    },
+    "layout_aliases": {
+        "LAYOUT_65_ansi_blocker": "LAYOUT_ansi_blocker",
+        "LAYOUT_65_ansi_blocker_split_bs": "LAYOUT_ansi_blocker_split_bs",
+        "LAYOUT_65_ansi_blocker_tsangan": "LAYOUT_ansi_blocker_tsangan",
+        "LAYOUT_65_ansi_blocker_tsangan_split_bs": "LAYOUT_ansi_blocker_tsangan_split_bs"
+    },
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk65/info.json:35]: - remove layout aliases
- replace with Community Layouts suport
```suggestion
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_split_bs", "65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs"],
```

File: keyboards/jaykeeb/jk65/info.json [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "Alt", "matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
+                {"label": "Ctrl", "matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "\u2190", "matrix": [4, 12], "x": 13, "y": 4},
+                {"label": "\u2193", "matrix": [4, 13], "x": 14, "y": 4},
+                {"label": "\u2192", "matrix": [4, 14], "x": 15, "y": 4}
+            ]
+        },
+        "LAYOUT_ansi_blocker": {
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk65/info.json:114]:
```suggestion
        "LAYOUT_65_ansi_blocker": {
```

File: keyboards/jaykeeb/jk65/info.json [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "Alt", "matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
+                {"label": "Ctrl", "matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "\u2190", "matrix": [4, 12], "x": 13, "y": 4},
+                {"label": "\u2193", "matrix": [4, 13], "x": 14, "y": 4},
+                {"label": "\u2192", "matrix": [4, 14], "x": 15, "y": 4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_split_bs": {
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk65/info.json:189]:
```suggestion
        "LAYOUT_65_ansi_blocker_split_bs": {
```

File: keyboards/jaykeeb/jk65/info.json [hunk 4]
```diff
@@ -0,0 +1,415 @@
...
+                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "Alt", "matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
+                {"label": "Ctrl", "matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "\u2190", "matrix": [4, 12], "x": 13, "y": 4},
+                {"label": "\u2193", "matrix": [4, 13], "x": 14, "y": 4},
+                {"label": "\u2192", "matrix": [4, 14], "x": 15, "y": 4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_tsangan": {
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk65/info.json:265]:
```suggestion
        "LAYOUT_65_ansi_blocker_tsangan": {
```

File: keyboards/jaykeeb/jk65/info.json [hunk 5]
```diff
@@ -0,0 +1,415 @@
...
+                {"label": "GUI", "matrix": [4, 1], "x": 1.5, "y": 4},
+                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
+                {"label": "Space", "matrix": [4, 6], "x": 4, "y": 4, "w": 7},
+                {"label": "Ctrl", "matrix": [4, 11], "x": 11, "y": 4, "w": 1.5},
+                {"label": "\u2190", "matrix": [4, 12], "x": 13, "y": 4},
+                {"label": "\u2193", "matrix": [4, 13], "x": 14, "y": 4},
+                {"label": "\u2192", "matrix": [4, 14], "x": 15, "y": 4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_tsangan_split_bs": {
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk65/info.json:339]:
```suggestion
        "LAYOUT_65_ansi_blocker_tsangan_split_bs": {
```

File: keyboards/jaykeeb/jk65/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_ansi_blocker(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+
+    [1] = LAYOUT_ansi_blocker(
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk65/keymaps/default/keymap.c:15]:
```suggestion
    [0] = LAYOUT_65_ansi_blocker(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
    ),

    [1] = LAYOUT_65_ansi_blocker(
```

File: keyboards/jaykeeb/jk65/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+
+    [1] = LAYOUT_ansi_blocker(
+        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,  _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PSCR, KC_SCRL, KC_PAUS, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+        _______,          _______, _______, _______, _______, _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______, _______, _______,
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk65/keymaps/via/keymap.c:19]:
```suggestion
    [0] = LAYOUT_all(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL,  KC_HOME,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
    ),

    [1] = LAYOUT_all(
        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PSCR, KC_SCRL, KC_PAUS, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______, _______, _______,
```

File: keyboards/jaykeeb/jk65/keymaps/via/rules.mk [hunk 8]
```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk65/keymaps/via/rules.mk:1]:
```suggestion
VIA_ENABLE = yes

```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by waffle87 — APPROVED** [review]

File: keyboards/jaykeeb/jk65/keymaps/via/rules.mk (hunk 8 above)
Comment (alabahuy) [reply keyboards/jaykeeb/jk65/keymaps/via/rules.mk]: done @dunk2k :)

**Comment by alabahuy** [comment]
any update @dunk2k ?

**Review by dunk2k — APPROVED** [review]

**Comment by alabahuy** [comment]
Any update @waffle87 ?

**Review by drashna — APPROVED** [review]

**Pull request merged**